babble-backend/controllers/posts-controller.js

```diff
@@ -0,0 +1,69 @@
+const db = require('../database/db');
+
+module.exports = {
+    getFeed,
+    createPost,
+    writeComment,
+    likePost,
+    savePost,
+    search,
+    getPost
+}
+
+/*************************************** */
+/* Get the userid with res.locals.userid */
+/*************************************** */
+
+/* Get the posts feed (all posts) */
+function getFeed (req, res) {
+    res.status(200);
+    res.json(db.posts.feed.filter(post => !post.isDeleted));
+}
+
+/* Create a post, req.body: title, body, tags (array of tags) */
+function createPost (req, res) {
+    // TODO
+    res.send('post created');
+}
+
+/* Make a comment, req.body: comment, query: post id (pid) */
```

Comment:
```suggestion
/* Make a comment, req.body: comment body, poster name, req.params: post id (pid) */
```

Reply:
Was the reasoning to not use query to get the post that we're not actually searching for the post, but just commenting on a post that we already have? If yes, that does make sense, but then similar changes should be made to the like and 
save-post routes.

Reply:
Yeah that and we already set up the endpoints to use params instead of queries lol. Don't think it would make too big of a difference but it would have been annoying to have to change all the existing routes so I just changed the documentation to reflect what we had instead.